src/app/globals.css

```diff
@@ -1,18 +1,13 @@
 @font-face {
   font-family: 'Lato';
-  src: url('assets/fonts/Lato-Light.ttf');
-  font-weight: 300;
+  src: url('./assets/fonts/Lato-Light.ttf');
+  src: url('./assets/fonts/Lato-Regular.ttf');
+  font-weight: 300, 400;
 }
 
 @font-face {
-  font-family: 'Lato';
-  src: url('assets/fonts/Lato-Regular.ttf');
-  font-weight: 400;
-}
-
-@font-face {
-  font-family: 'Permanent Marker';
-  src: url('assets/fonts/PermanentMarker-Regular.tff');
+  font-family: 'Permanentmarker';
+  src: url('./assets/fonts/PermanentMarker-Regular.tff');
```

Comment:
LGTM

Reply:
only this is weird because Permanent Marker is not the font that appears in storybook